migrations/245_group_role_tables__create.sql

```diff
@@ -0,0 +1,38 @@
+-- Create table and index for group roles
+CREATE TABLE IF NOT EXISTS group_roles (
+    id BIGSERIAL PRIMARY KEY,
+    role_name TEXT NOT NULL,
+    assessment_id BIGINT NOT NULL,
+    minimum INT DEFAULT 0,
+    maximum INT DEFAULT 4,
+    can_assign_roles_at_start BOOLEAN DEFAULT FALSE,
+    can_assign_roles_during_assessment BOOLEAN DEFAULT FALSE,
+    can_submit_assessment BOOLEAN DEFAULT TRUE,
+    CONSTRAINT group_roles_assessment_id_fkey
+        FOREIGN KEY (assessment_id)
+            REFERENCES assessments(id) ON UPDATE CASCADE ON DELETE CASCADE
```

Comment:
Instead of separately declaring a foreign key constraint, you should do it when you create the column:

```suggestion
    assessment_id BIGINT REFERENCES assessments(id) ON UPDATE CASCADE ON DELETE CASCADE,
    minimum INT DEFAULT 0,
    maximum INT DEFAULT 4,
    can_assign_roles_at_start BOOLEAN DEFAULT FALSE,
    can_assign_roles_during_assessment BOOLEAN DEFAULT FALSE,
    can_submit_assessment BOOLEAN DEFAULT TRUE
```